backend/JavaWebService/src/test/java/JobSpecificationControllerTest.java

```diff
@@ -0,0 +1,33 @@
+import com.kainos.ea.resources.Band;
+import com.kainos.ea.resources.JobSpecification;
+import com.kainos.ea.resources.JobSpecificationService;
+import com.kainos.ea.resources.JobSpecificationController;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class JobSpecificationControllerTest {
+    @Mock
+    private JobSpecificationService mockService;
+
+    @Test
+    public void controllerReturnsSameSpecificationAsServiceProvides(){
+        //arrange
+        JobSpecificationController controller = new JobSpecificationController(mockService);
+        JobSpecification jobSpecificationObj = new JobSpecification();
```

Comment:
The `Obj` suffix isn't helpful - they're all objects. The distinction is just that one of them is the expected object and the other is the actual object, so I'd suggest renaming this accordingly.